src/components/attendee/InputSelect/InputSelect.tsx

```diff
@@ -18,70 +18,83 @@ interface InputSelectProps
     ContainerClassName {
   inputClassName?: string;
   errorTextClassName?: string;
-  error?: FieldError;
+  errorSelect?: FieldError;
+  errorInput?: FieldError;
   inputName: string;
   selectName: string;
   inputPlaceholder?: string;
   selectPlaceholder?: string;
   register: UseFormRegister<AnyForm>;
-  inputRules: RegisterOptions;
-  selectRules: RegisterOptions;
+  inputRules?: RegisterOptions;
+  selectRules?: RegisterOptions;
 }
 
 export const InputSelect: React.FC<InputSelectProps> = ({
   containerClassName,
   inputClassName,
   errorTextClassName,
-  error,
+  errorSelect,
+  errorInput,
   inputName,
   selectName,
   inputPlaceholder,
   selectPlaceholder,
   register,
-  inputRules,
-  selectRules,
+  inputRules = {},
+  selectRules = {},
```

Comment:
[NOTE]
I'm not sure about this one. Usually there should be no error, but in case there is one, do we need the other one?

Do we return `error: FIeldError | FieldError[]` or `error: FIeldError | Record<string,FieldError>` instead of having twice the props?

Just something to think about for later date.

Reply:
[OOS] There are some seed sparkle flavored errors that might work with this multiple errors returned thing, if for example there is `error: SparkleErrors` which is an error that has an array of `cause` errors inside.

Improvement for other times I guess, so can we please add a `@debt` comment here so we can return to it?

Reply:
added a debt comment to possibly unite error objects into a single entity [614df69](https://github.com/sparkletown/sparkle/pull/2926/commits/614df691194b102f72660673698d0af99c340241)